### PR TITLE
Partners page

### PR DESCRIPTION
**Pull request by sikakkar**

### CODE REVIEW

**Comment by linderr** [comment]
@sikakkar hi! Thank you so much for doing this page :) Great work ! Just a few tweaks:

1. All of these tiles should link out to the respective page --- we have listed all the articles (in press section) and homepages (partners and collaborators) for you in the Google Sheets!
Please see Column F for the URLS
2. BlogTO and eTalk/CTV should be both under "In Press" --- there is a legend for colours on the Google sheets. Blue = sponsor, Red = grassroot collaborator, Grey = In Press. Thank you otherwise it looks great!!!

**Comment by linderr** [comment]
Just merged Smartling translations. If code is fine, ready to merge.

**Comment by linderr** [comment]
Above edits still need to be changed before merging ^^

**Comment by sikakkar** [comment]
Okay, all done on the above. Should be good to go now.

**Comment by linderr** [comment]
I'll wait for Rachel's deployment to view and help you test @sikakkar thank you!

**Comment by linderr** [comment]
@sikakkar this is your deployment: https://findthemasks-partners-p-nq08t4.herokuapp.com/partners
Is this what you expect?? I still cannot click the links, sorry

File: sass/style.css [hunk 1]
```diff
@@ -317,6 +317,29 @@ h3.city {
   opacity: 0.4;
 }
 
+/*********************
+  * PARTNERS STYLES *
+  *********************/
+h1.page-title {
+  text-transform: uppercase;
```
Comment (nburt) [inline sass/style.css:324]: Did we vet this for other languages? Not sure if we want text-transform for all languages / countries

File: server.js [hunk 2]
```diff
@@ -158,14 +158,25 @@ router.get('/blog/2020-04-21-data-insights', (req, res) => {
...
-    ogLocale:  formatFbLocale(res.locals.locale),
+    ogLocale: formatFbLocale(res.locals.locale),
     ogTitle: res.locals.banana.i18n('ftm-about-us-og-title'),
     ogUrl: `http://${req.hostname}${req.originalUrl}`,
     ogDescription: res.locals.banana.i18n('ftm-default-og-description'),
     version: herokuVersion
   });
 });
 
+router.get(['/partners', '/partners.html'], (req, res) => {
```
Comment (nburt) [inline server.js:169]: nit: we don't need the `/partners.html` here. The give/whoweare/index are legacy for backwards compatibility from before this was an express server

File: views/partials/top_nav.handlebars [hunk 3]
```diff
@@ -55,6 +55,11 @@
           FAQ
         </a>
       </li>
+      <li class="nav-item active">
+        <a class="nav-link i18n" data-i18n="ftm-nav-partners" href="javascript:;" onclick="window.location='partners'+window.location.search;">
```
Comment (nburt) [inline views/partials/top_nav.handlebars:59]: I think this point was brought up in slack earlier for the whoweare page but this isn't great for crawlers. Can you either fix (set href with current country + locale in render-nav.js) or create a new issue for that?

File: views/partners.handlebars [hunk 4]
```diff
@@ -0,0 +1,117 @@
+<main class="content container py-5">
+  <h1 class="page-title">Sponsors, Collaborators &amp; Media</h1>
```
Comment (nburt) [inline views/partners.handlebars:2]: missing i18n

File: views/partners.handlebars [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+  <p class="addTopMargin i18n" data-i18n="ftm-partners-info">We thank our sponsors and collaborators for 
+    their support of our mission to connect health care workers with the PPE they need to stay safe. 
+    To partner with us, reach out to email@email.com.</p>
+
+  <h3 class="page-subhead">Global Sponsors</h3>
```
Comment (nburt) [inline views/partners.handlebars:8]: missing i18n

File: views/partners.handlebars [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+      <img src="images/logos-CA/The Home Front.jpg" alt="The Home Front" width="140">
+    </a>
+    <a href="http://www.trackppe.com/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Track PPE.png" alt="Track PPE" width="140">
+    </a>
+    <a href="www.theppedrive.com" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/thePPEDrive.png" alt="thePPEDrive" width="140">
+    </d>
+    <a href="www.fb.com/groups/VolunteersinOntarioFindingPPEforHospitals/" target="_blank" class="partner-logo-container">
+      <img src="images/logos-CA/Volunteers in Ontario Finding PPE  for Hospitals.jpg" alt="Volunteers in Ontario Finding PPE  for Hospitals.jpg" width="140">
```
Comment (nburt) [inline views/partners.handlebars:82]: you can do i18n for alt attrs as well. `class="i18n" data-i18n="[alt]ftm-my-key"` should work

File: views/partials/top_nav.handlebars (hunk 3 above)
Comment (saikatbhadra) [reply views/partials/top_nav.handlebars:59]: let's just create a new issue to fix all of those

**Comment by saikatbhadra** [comment]
Looks good except for some minor i18n stuff

**Comment by saikatbhadra** [comment]
very purty

**Comment by linderr** [comment]
Before merging --- point 5. 
Sorry!! Order of funding/collaborative effort descending order from Sheets

**Comment by saikatbhadra** [comment]
> Before merging --- point 5.
> Sorry!! Order of funding/collaborative effort descending order from Sheets -- some collaborators gave a looot of data, or helped to prepare that data, hand-entry etc etc.
> 
> > 1. Also the order matters for collaborators! We listed on Google Sheets from descending order of importance for collaborators. Just follow the Google Sheets file.

What google sheets are you referring to? The order looks correct based on the figma design but I don't see any link to any sheet here.

File: views/partners.handlebars (hunk 6 above)
Comment (sikakkar) [reply views/partners.handlebars:82]: Since most of these are names of organizations, does it make sense to translate them?

**Comment by sikakkar** [comment]
@linderr order updated. Links are working for me!
@nburt i18n added, except alt tags. I don't see smartling though, is it running?

**Comment by linderr** [comment]
@sikakkar awesome job. Yes! Just clicked through all -- no buggies, yay!
sorry, last minute logo change by collaborator: Shield Net --- they want to go with the UW-Madison logo. I have updated the two places for you in Google Drive: US and CA regional folders. I kept the same file name (Shield Net.png) - not sure if you have to change anything on the back end but FYI!

**Comment by sikakkar** [comment]
@linderr updated shield net logo

**Comment by linderr** [comment]
YAYYYY @sikakkar

**Pull request merged**